src/main/java/racingcar/pattern/RandomNumberGenerator.java

```diff
@@ -0,0 +1,14 @@
+package racingcar.pattern;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+public class RandomNumberGenerator implements NumberGenerator {
+
+    private static final Random random = new SecureRandom();
+
+    @Override
+    public int generate() {
+        return random.nextInt(MAX +1);
```

Comment:
```suggestion
        return random.nextInt(MAX + 1);
```

코드 정렬은 사소하지만 매우 중요하다고 생각해요
IDE의 자동정렬을 활용해보는건 어떨까요?